client/post-editor/editor-revisions-list/index.jsx

```diff
@@ -57,12 +56,44 @@ class EditorRevisionsList extends PureComponent {
 	componentDidMount() {
 		// Make sure that scroll position in the editor is not preserved.
 		window.scrollTo( 0, 0 );
+
+		KeyboardShortcuts.on( 'move-selection-up', this.selectNextRevision );
+		KeyboardShortcuts.on( 'move-selection-down', this.selectPreviousRevision );
+	}
+
+	componentWillUnmount() {
+		KeyboardShortcuts.off( 'move-selection-up', this.selectNextRevision );
+		KeyboardShortcuts.off( 'move-selection-down', this.selectPreviousRevision );
 	}
 
 	componentDidUpdate() {
 		this.trySelectingRevision();
 	}
 
+	selectNextRevision = () => {
```

Comment:
I've opted for a bit of repetition here in order to maintain readability & straight forwardness.
The alternative would be to create a method that allows `1` or `-1` to be passed and creating 2 new functions from it using `_.partial` or `bind`. Happy to go that route if others prefer :)